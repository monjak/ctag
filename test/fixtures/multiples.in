Oblivion ring
Oblivion ring
Oblivion ring
Oblivion ring
Mind spring
Fabricate
Fabricate
Relentless rats
Relentless rats
