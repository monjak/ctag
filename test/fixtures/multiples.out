[c]Oblivion ring[/c]
[c]Oblivion ring[/c]
[c]Oblivion ring[/c]
[c]Oblivion ring[/c]
[c]Mind spring[/c]
[c]Fabricate[/c]
[c]Fabricate[/c]
[c]Relentless rats[/c]
[c]Relentless rats[/c]
